test/integration/narrow/routes/capture.test.js

```diff
@@ -74,11 +74,11 @@ describe('Capture test', () => {
     })
 
     test.each([
+      { frn: 1234567890, statusCode: 200 },
```

Comment:
FRN is a 10 digit number, therefore this should return 200.

Post casting to String enabled this, is this right?